### PR TITLE
feat: added chart for AWS Inferenria operator

### PR DESCRIPTION
**Pull request by debajyoti-truefoundry**

### CODE REVIEW

File: charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}-neuron-device-plugin
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      name: {{ .Release.Name }}-neuron-device-plugin-ds
+  updateStrategy:
+    type: RollingUpdate
```
Comment (chiragjn) [inline charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml:70]: We'll have to see if this safe when some workload is already running

Comment (debajyoti-truefoundry) [reply charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml:70]: This is coming straight from https://github.com/aws-neuron/aws-neuron-sdk/blob/e0ef8a1a780ee798e7f01fe94f1235d571e211c6/src/k8/k8s-neuron-device-plugin.yml#L1

File: charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        {{- toYaml (index .Values "devicePlugin" "affinity") | nindent 8 }}
+      containers:
+      - image: .Values.devicePlugin.image
+        imagePullPolicy: Always
+        name: neuron-device-plugin
+        resources:
+          {{- toYaml (index .Values "devicePlugin" "resources") | nindent 10 }}
+        env:
+        - name: KUBECONFIG
+          value: /etc/kubernetes/kubelet.conf
```
Comment (chiragjn) [inline charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml:94]: Weird, is this part of the image itself, don't see it as a mount

File: charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml (hunk 1 above)
Comment (chiragjn) [reply charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml:70]: Yeah, I checked, unfortunately some defaults are not good for actual use cases

**Review by chiragjn — APPROVED** [review]

File: charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml (hunk 1 above)
Comment (debajyoti-truefoundry) [reply charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml:70]: I triggered a rollout. It seems to be working fine.

File: charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml (hunk 2 above)
Comment (debajyoti-truefoundry) [reply charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml]: Removed.

**Pull request merged**